### PR TITLE
Minor edits to docs

### PR DESCRIPTION
**Pull request by robelgeda**
Update information in the readme file.

### CODE REVIEW

**Comment by robelgeda** [comment]
@k-gilbert it should be good now?

File: README.md [hunk 1]
```diff
@@ -224,8 +224,11 @@ This will create a new folder called `wfirst-tools` containing this README and a
 
 The STScI helpdesk at help@stsci.edu is available for members of the WFIRST scientific community. For issues with WebbPSF, we prefer that you report your issues in the GitHub issue tracker for the speediest response: https://github.com/spacetelescope/webbpsf/issues (choose the green "New Issue" button after logging in).
```
Comment (k-gilbert) [inline README.md]:
```suggestion
Users are encouraged to address questions, suggestions, and bug reports to help@stsci.edu with "Pandeia-WFIRST question" in the subject line. The message will be directed to the appropriate members of the Pandeia-WFIRST team at STScI.  For issues with WebbPSF, we prefer that you report your issues in the GitHub issue tracker for the speediest response: https://github.com/spacetelescope/webbpsf/issues (choose the green "New Issue" button after logging in).
```

File: README.md [hunk 2]
```diff
@@ -224,8 +224,11 @@ This will create a new folder called `wfirst-tools` containing this README and a
...
 
+Users are encouraged to address questions, suggestions, and bug reports to help@stsci.edu with "Pandeia-WFIRST question" in the subject line. The message will be directed to the appropriate members of the Pandeia-WFIRST team at STScI.
```
Comment (k-gilbert) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -94,9 +94,9 @@ And then run `./run.sh` again.
 
 ## Install the simulation tools locally
 
-The WebbPSF point-spread function model and Pandeia exposure time calculator engine are currently available for local installation by members of the science community. The required packages are distributed as part of Astroconda, a suite of astronomy-focused software packages for use with the [conda](https://conda.io/docs/) package manager for macOS and Linux.
+STIPS, the Space Telescope Image Product Simulator, is available at https://github.com/spacetelescope/STScI-STIPS.
```
Comment (k-gilbert) [inline README.md]: I actually meant to move the new STIPS sentence to the end of the very first paragraph in the README- which would become (with quite a few modifications I'm making here, while we are at it):

The WFIRST team at STScI has developed an exposure time calculator, a PSF model, and an image simulator for the science community to plan how they will use WFIRST. These tools are available separately as the Pandeia exposure time calculator engine, the WebbPSF point spread function modeling package, and the Space Telescope Image Product Simulator (STIPS).   Comprehensive setup documentation for local installation as well as tutorials for Pandeia and WebbPSF are provided here.  STIPS is available at https://github.com/spacetelescope/STScI-STIPS.  High level overviews of the functionality of the tools are available on [STScI's WFIRST website](http://www.stsci.edu/scientific-community/wide-field-infrared-survey-telescope/science-planning-toolbox).

**Comment by k-gilbert** [comment]
@robelgeda, those changes aren't quite what I was looking for, but I just learned how to insert comments (and code change suggestions) in the "Files changed" tab, so hopefully my intention will be clearer now.

**Comment by k-gilbert** [comment]
@robelgeda, those changes aren't quite what I was looking for, but I just learned how to insert comments (and code change suggestions) in the "Files changed" tab, so hopefully my intention will be clearer now.

**Comment by robelgeda** [comment]
@k-gilbert, I made the edits and committed the one's you recommended.

**Comment by k-gilbert** [comment]
@robelgeda, that looks good to go.  Please go ahead and merge into spacetelescope:master.  Thanks!

**Pull request merged**